apps/admin-gui/src/assets/i18n/en.json

```
@@ -2148,6 +2151,17 @@
       "BEGIN_SEARCH": "Search for members you would like to sponsor",
       "SUCCESS": "Members successfully sponsored"
     },
+    "COPY_SPONSORED_MEMBERS": {
+      "TITLE": "Copy sponsored members",
+      "CANCEL": "Cancel",
+      "SPONSOR": "Copy",
+      "SPONSORED_MEMBERS": "Select sponsored members to copy",
+      "SPONSORED_MEMBERS_INFO": "Members already sponsored by the destination sponsor will not be affected.",
```

We are mixing the terms a little bit the selections say source and target. I don't mind which one we use but stay consistent.
```suggestion
      "SPONSORED_MEMBERS_INFO": "Members already sponsored by the target sponsor will not be affected.",
```